src/SecureStore.BeyondTrust/UiPath.Orchestrator.BeyondTrust-DynamicSystem-ReadOnly/UiPath.Orchestrator.BeyondTrust-DynamicSystem-ReadOnly.csproj

```diff
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <RootNamespace>UiPath.Orchestrator.BeyondTrust_ReadOnly</RootNamespace>
```

Comment:
Let's not use `-` and `_` in either namespace or project name, that is uncommon in .net. As said above, we should have a single project named `UiPath.Orchestrator.BeyondTrust` with the default root namespace.

Reply:
Fixed when merging projects